hal/armv7a/zynq7000/types.h

```diff
@@ -52,8 +52,18 @@ typedef unsigned int size_t;
 typedef int ssize_t;
 typedef unsigned long long time_t;
 
+#define MPU_REGION_NB 16
```

Comment:
`zynq7000` doesn't have MPU. 
Please remove this define

Reply:
Ooops,  `git mergetool` after directory reorganization. Reverted any changes in `types.h`.